app/views/govuk_publishing_components/components/_chevron_banner.html.erb

```diff
@@ -7,17 +7,20 @@
   data_attributes[:module] = 'track-click' if data_attributes.any?
 %>
 <% if href && text %>
-  <% #This assumes the application calling the component supports passing :extra_headers into the layout. %>
-  <% #If not, add the following to your application layout file: `yield :extra_headers` %>
-  <% content_for :extra_headers do %>
-    <link rel="preload" as="image" href="<%= asset_path 'govuk_publishing_components/chevron-banner/chevron-banner-hover.svg' %>" type="image/svg+xml">
-    <link rel="preload" as="image" href="<%= asset_path 'govuk_publishing_components/chevron-banner/chevron-banner-hover-border.svg' %>" type="image/svg+xml">
-    <link rel="preload" as="image" href="<%= asset_path 'govuk_publishing_components/chevron-banner/chevron-banner-focus.svg' %>" type="image/svg+xml">
-  <% end %>
-
   <div class="gem-c-chevron-banner <%= "gem-c-chevron-banner--hover-border" if hover_border %>">
-    <div class="gem-c-chevron-banner__link-container">
-      <%= link_to(text, href, class: "gem-c-chevron-banner__link govuk-link", data: data_attributes) %>
-    </div>
+    <%= link_to href, class: "gem-c-chevron-banner__link", data: data_attributes, role: "text" do %>
+      <span class="gem-c-chevron-banner__text"><%= text %></span>
+      <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 -.2 175 109.5" class="gem-c-chevron-banner__chevron" preserveAspectRatio="xMidYMid meet">
+        <defs>
+          <path id="a" d="M0 0h93v109H0z"></path>
```

Comment:
I _think_ this is causing the aXe error in the component guide. Although even if it isn't, I don't think it's needed?

Reply:
I agree, the SVG could be simplified to something like the code below or even more
```
      <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 175 109" class="gem-c-chevron-banner__chevron" preserveAspectRatio="xMidYMid meet">
        <path class="gem-c-chevron-banner__chevron-point" fill="#E61E32" fill-rule="nonzero" d="M91.611 51.664c1.0371 1.1432 1.0375 2.8869.000802 4.0305l-44.991 49.634c-.37092.40919-.33989 1.0416.069307 1.4125.18394.16674.42334.25909.6716.25909h36.892c.55978 0 1.0939-.2346 1.4727-.6468l46.534-50.643c1.0549-1.148 1.0546-2.9127-.000709-4.0604L85.3617.64689c-.37873-.41188-.91268-.64628-1.4722-.64628h-36.891c-.55228 0-1 .44772-1 1 0 .24841.092454.48792.25936.6719l45.352 49.992z" transform="translate(0 1)"></path>
        <path class="gem-c-chevron-banner__chevron-base" fill="#E61E32" fill-rule="nonzero" d="M-37.997 1l.35768 107h40.408c.55978 0 1.0939-.2346 1.4727-.6468l46.534-50.643c1.0549-1.148 1.0546-2.9127-.0007095-4.0604l-46.898-51.003c-.37873-.41188-.91268-.64628-1.4722-.64628h-40.402z" vector-effect="non-scaling-stroke"></path>
      </svg>

```

Will have a think about how to avoid that overflow though..